.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: NPM Install
+        run: npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Run tests
+        run: |
+          npm ci
+          npm test
+  update-api-definitions-changelog:
+    runs-on: ubuntu-latest
+    needs: [test]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: Update Changelog
+        run: |
+          bash scripts/updateApiDefinitions.sh "${{ github.event.inputs.changeLog }}" ${{ github.event.inputs.versionType }}
+  release:
+    runs-on: ubuntu-latest
+    needs: [update-api-definitions-changelog]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: Run git and npm update
+        run: |
+          git pull
+          npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Semantic Release runs
+        run: |
+          npm install --save-dev @semantic-release/changelog @semantic-release/git
+          npm ci
+          npx semantic-release -t \${version}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  update-release:
+    runs-on: ubuntu-latest
+    needs: [ release ]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: Getting tag
+        id: get-tag
+        run: |
+          git fetch --tags
+          echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
+      - run: echo "${{steps.get-tag.outputs.TAG_NAME}}"
+      - name: Update release
+        uses: tubone24/update_release@v1.2.0
```

Comment:
This action seems to be community contributed by an independent developer, might lead to future support/bugs handling. Any possibility of any alternative option than this? 

https://github.com/tubone24/update_release

Reply:
Yeah, Agreed. But I was not able to find an action that is developed by GitHub or Publisher verified. I will check once again though. If you see any verified action please let me know.

Reply:
Just doubt the community support for such actions. I guess @kridai  also faced this issue which can be explored further..